subsys/net/lib/nrf_cloud/Kconfig.nrf_cloud_mqtt

```diff
@@ -68,6 +69,7 @@ config NRF_CLOUD_SEND_NONBLOCKING
 
 config NRF_CLOUD_SEND_TIMEOUT
 	bool "Send data with socket timeout"
+	depends on !BOARD_QEMU_X86
```

Comment:
Why? `SO_SNDTIMEO` is supported by the native stack. There's a Kconfig knob to enable it though (`CONFIG_NET_CONTEXT_SNDTIMEO`).

Reply:
Wasn't aware of this, will remove the change.